### ISSUE TITLE
You library works well! Thank you!

### ISSUE THREAD

**Issue by MatthewMerritt** [p-sun/iOSMultiplayerTemplate#1]
I have added a little functionality that you may find useful.

* I updated the MCBrowser code to allow macOS to use it.
* I added var from: MCPeerID? to P2PSyncedObservable and P2PSynced so we can easily view who sent an object.

I would be happy to share these with you if you would like. Nothing big, but helpful in my situation.

**Comment by p-sun** [comment]
I'm glad to hear that my library helped. Thanks for the comment!

**Comment by p-sun** [comment]
Yeah, I'd be curious to see what you've changed, especially for MacOS. Feel free to send me a zip!

**Comment by MatthewMerritt** [comment]
Ok, I have P2PKit completed. I can send that or I can work on making P2PKitDemo macOS compatible also and send them together. Whichever you would like.

**Comment by p-sun** [comment]
Just P2PKit would be neat. No worries if you’re busy. :)

**Comment by MatthewMerritt** [comment]
Ok, so I have most of the demo ported to macOS. I will need another day or 2 for the actual game part, but the Debug Tab is all done.

**Comment by p-sun** [comment]
As is perfectly fine. No worries about porting the whole thing. Feel free to message me on Twitter @_PaigeSun if you'd like too.

**Comment by MatthewMerritt** [comment]
It’s almost done, just gotta work on gestures to move the pucks around. All else is working. I don’t have twitter believe it or not lolSent from my iPhoneOn Jul 13, 2024, at 8:07 AM, Paige Sun ***@***.***> wrote:﻿
As is perfectly fine. No worries about porting the whole thing. Feel free to message me on Twitter @_PaigeSun if you'd like too.

**Comment by MatthewMerritt** [comment]
Not sure it this will work, but I’m ready to send you a zip. There is a bug, that all the mallets that are not chosen disappear when 1 is. But other than that it seems to work.

I have added a few files in the in both folders and tried to make some comments here and there. If you want you can use it how ever you choose. If you have questions about what or why I have done something I would be happy to chat about it. Also if you find what the bug above is and fix it I would be curious to know.

Also, if you want it and feel there is value let me know and I will spend more time cleaning up my code and documenting.

Thanks,
Matt
￼

**Comment by p-sun** [comment]
Hi! Feel free to send it to paige.sun.dev@gmail.com.

**Comment by MatthewMerritt** [comment]
Hello, did you find any of the code useful?
